spec/hardware/gpu.fmf

```diff
@@ -1,13 +1,38 @@
-summary:
-    Choose system according to the available graphics processing unit (gpu)
+summary: |
+    Select or provision a guest with a given GPU properties.
+
+description: |
+    .. code-block::
+
+       gpu:
+           # String, a network device name.
+           device-name: "A Generic Graphics Card"
+
+           # Number or string, a network device ID.
+           device: 1234|"> 1234"
+
+           # String, a name of the device vendor.
+           vendor-name: "A Well-Known GPU Manufacturer"
+
+           # Number or string, an ID of the device vendor.
+           vendor: 1234|"> 1234"
+
+    .. versionadded:: 1.29
+
 example:
   - |
-    # GPU-related stuff grouped together.
+    # Request a guest with particular GPU
+    gpu:
+        # By the device name...
+        device-name: C51 [GeForce 6150 LE]
+        # ... or by its ID.
+        device: 0241
+        driver: nouveau
```

Comment:
The `driver` key is not provided in the spec above. Should be added there or removed from here?

Reply:
Right. It should be added: it's part of `device` block, and since `gpu` (and `network` too) are supersets of `device`, they can support it right away. So, adding `driver` to both these requirements, including their schemas.

Reply:
Fixed in b42f566a.